docs/user/bigtable/dataproc.rst

```diff
@@ -0,0 +1,84 @@
+GeoMesa Spark SQL on Google Cloud Dataproc
+==========================================================
+
+GeoMesa can run Spark SQL with Bigtable as the underlying datastore.  In order to set up Spark SQL,
+you will need to launch a Google Cloud Dataproc cluster.  First, you will need to install the Google
+Cloud SDK command line tools.  Instructions for doing so can be found `here <https://cloud.google.com/sdk/downloads>`.
```

Comment:
RST requires a trailing underscore for this link to render properly:

     `here <https://cloud.google.com/sdk/downloads>`_.